tests/components/overkiz/test_config_flow.py

```diff
@@ -204,3 +204,47 @@ async def test_dhcp_flow_already_configured(hass: HomeAssistant) -> None:
 
     assert result["type"] == data_entry_flow.RESULT_TYPE_ABORT
     assert result["reason"] == "already_configured"
+
+
+async def test_reauth_success(hass):
+    """Test reauthentication flow."""
+    with patch(
```

Comment:
Is `pyoverkiz.client.OverkizClient.login` actually getting invoked anywhere here?  This looks like its not actually running init, but the test invokes reauth as if this did throw from init. Apologies if i missed it, but this seems like it can be removed.

Reply:
Fixed in dc1c701e3b24748b483ef0ac0b2e836a63356ec0.